From: Gordon Ball <gordon@chronitis.net>
Date: Mon, 18 Dec 2017 16:26:25 +0000
Subject: Disable sphinxcontrib_github_alt

Disables the sphinx extension sphinxcontrib_github_alt (unpackaged)
during the documentation build.
---
 docs/conf.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/docs/conf.py b/docs/conf.py
index 23c112a..33502ef 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -29,11 +29,9 @@ import shutil
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    'myst_parser',
     'sphinx.ext.autodoc',
     'sphinx.ext.intersphinx',
     'sphinx.ext.napoleon',
-    'sphinxcontrib_github_alt',
 ]
 
 myst_enable_extensions = ["html_image"]
